static/coffee/controllers.coffee

```diff
@@ -573,12 +588,12 @@ controllers.controller('CaseController', ['$scope', '$window', '$timeout', 'Case
   #----------------------------------------------------------------------------
 
   $scope.onWatch = () ->
-    UtilsService.confirmModal("Receive notifications for activity in this case?").then(() ->
+    UtilsService.confirmModal("Receive alerts for activity in this case?").then(() ->
```

Comment:
What's the reason for this change? Seems to me alerts and notifications are different things. 

I think I might have mentioned this on the dev list but I think at some point it would make sense to surface notifications in the UI. Should also ensure that pods have the ability to generate notifications and control how they are rendered as emails.

Also can we replace these calls with `ModalService.confirm(..)` calls so we're certain that new service can replicate the existing functionality, and we don't have two confirm modal methods.